README.rst

```diff
@@ -23,69 +23,67 @@ Kagi
 
 Kagi provides support for FIDO WebAuthn security keys and TOTP tokens in Django.
 
-Kagi is a working proof-of-concept and isn't yet production ready. There are
-many TODOs sprinkled around the code that should be fixed before relying on it.
+Kagi is a working proof-of-concept and isn’t yet production-ready. There may be
```

Comment:
I would remove this sentence altogether. Hence there are no todo left in the code.

Reply:
Since the last remaining TODO in the base template will be resolved soon, I will remove this line and then merge this PR. Thanks!